features/old/applications/providers/create_application.feature

```diff
@@ -78,6 +78,7 @@ Feature: Create application
    Scenario: The service of the selected application plans hasn´t service plan
      Given a service "second" of provider "foo.example.com"
        And a published application plan "second_app_plan" of service "second" of provider "foo.example.com"
+       And the service "second" does not have service plan
```

Comment:
I don't understand. Normally there would be a (default) service plan, how would it be possible to create an app against a service that doesn't have a (default) service plan?

Reply:
This test is to show some link to create a service plan if it happens it is deleted

![image](https://user-images.githubusercontent.com/64276/64427560-24b75580-d0ba-11e9-8fc0-d1c596d980e6.png)

Anyway clicking on this link leads to a 404
We can either remove the test and also the feature or fix it